indico/modules/rb_new/client/js/util.jsx

```diff
@@ -112,6 +117,24 @@ export function boolStateField(name) {
     };
 }
 
+/**
+ * Defines a query string state field that is only added to the query
+ * string if the value doesn't match the default.
+ * @param {string} name - Name of the state field.
+ * @param defaultValue - The default value that will not go to the query string.
+ */
+export function defaultStateField(name, defaultValue) {
+    return {
+        serialize: state => {
+            const value = _.get(state, name, null);
```

Comment:
why do you have to use `_.get` here? isn't it enough to use `return state[name] === defaultValue ? null : value`? Does it prevent from cases where `defaultValue` is `undefined`?

Reply:
No, the state field can use dot notation